models/dodo/ethereum/dodo_ethereum_schema.yml

```diff
@@ -96,4 +108,98 @@ models:
         description: ""
       - &evt_index
         name: evt_index
-        description: "Index of the corresponding trade event"
\ No newline at end of file
+        description: "Index of the corresponding trade event"
+
+  - name: dodo_pools_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: dodo_pools
+      contributors: scoffie, owen05
+    config:
+      tags: ['ethereum','dodo',"pools", 'trades', 'dex']
+    description: >
+        DodoV1 && DODOV2 pools' trades on Ethereum.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+    columns:
+      - *blockchain
+      - *project
+      - *version
+      - *block_date
+      - *block_time
+      - *token_bought_symbol
+      - *token_sold_symbol
+      - *token_pair
+      - *token_bought_amount
+      - *token_sold_amount
+      - *token_bought_amount_raw
+      - *token_sold_amount_raw
+      - *amount_usd
+      - *token_bought_address
+      - *token_sold_address
+      - *taker
+      - *maker
+      - *project_contract_address
+      - *tx_hash
+      - *tx_from
+      - *tx_to
+      - *trace_address
+      - *evt_index
+
+  - name: dodo_aggregator_ethereum_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: dodo_aggregator
+      contributors: scoffie, owen05
+    config:
+      tags: ['ethereum','dodo',"aggregator", 'trades', 'dex']
+    description: >
+        DODO Aggregation on Ethereum.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: ethereum
+          project: DODO
+          version: 0
```

Comment:
@owen05 I added an extra test here, as by definition, records with version 0 is for aggregator trades. You can check if the seeds in the seed file match the expectation.

Reply:
Thanks. The seeds match the expectation